conftest.py

```diff
@@ -20,7 +20,7 @@ def pytest_addoption(parser):
     parser.addoption("--browser-channel", default=None)
     parser.addoption("--device", default=None)
     parser.addoption("--slowmo", type=int, default=200)
-    parser.addoption("--headed", action="store_true", default=False)
+    parser.addoption("--headed", action="store_true", default=True)
```

Comment:
I don't think we'd want this by default because we will want it off in CI environments.